spec/support/stubbed_vacuum.rb

```diff
@@ -0,0 +1,68 @@
+class StubbedVacuum
+  def initialize(details)
+    @details = details
+  end
+
+  def associate_tag=(something)
+    #noop
+  end
+
+  def item_lookup(query)
+    VacuumResponse.new(@details)
+  end
+end
+
+class VacuumResponse
+  def initialize(details)
+    @details = details
+  end
+
+  def to_h
+    {"ItemLookupResponse"=>
+     {"OperationRequest"=>
+      {"HTTPHeaders"=>{"Header"=>{"Name"=>"UserAgent", "Value"=>"Jeff/1.5.1 (Language=Ruby; ares.local)"}},
+       "RequestId"=>"a8279e28-02d2-4703-b4bc-669ab41f4d02",
+       "Arguments"=>
+      {"Argument"=>
+       [
+         {"Name"=>"AWSAccessKeyId", "Value"=>"AKIAINQWKZ76LYIIEWVQ"},
```

Comment:
Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.